ui/StatusQ/src/StatusQ/Core/Utils/AmountsArithmetic.qml

```diff
@@ -140,6 +140,16 @@ QtObject {
         return amount1.plus(amount2)
     }
 
+    /*!
+      \qmlmethod AmountsArithmetic::sub(amount1, amount2)
+      \brief Returns a Big number whose value is the subtraction of amount2 from amount1.
+     */
+    function sub(amount1, amount2) {
+        console.assert(amount1 instanceof Big.Big)
+        console.assert(amount2 instanceof Big.Big)
```

Comment:
Technically, the `amount2` doesn't have to be a `Big`, you can subtract a regular JS Number too

Reply:
Thanks for the heads up, I fixed it like this `console.assert(amount2 instanceof Big.Big || Number.isInteger(amount2))`